apidoc/Titanium/UI/TextField.yml

```diff
@@ -573,6 +573,14 @@ events:
         summary: Value of this text area.
         type: String
 
+  - name: keyPressed
```

Comment:
It is `keypressed`: http://docs.appcelerator.com/platform/latest/#!/api/Titanium.UI.TextField-event-keypressed

Also, in Android the event should already by there in every API, so we just need to add the platforms to the related docs.